README.md

```diff
@@ -1,11 +1,15 @@
-## What is oah-bes-vm 
+## What is oah-bes-vm?
 
 A substrate that brings up bundles of BeS & other tools to perform Red Team & Blue Team activity.
 
 ## Who are the end users?
 
 RT & BT activist of BeS community.
```

Comment:
@pramit-d link to the definition of RT & BT , everyone need not know what RT & BT is

Reply:
@panickervinod added RT BT definition.